### DISCUSSION TITLE
Instagram

### DISCUSSION THREAD

**Discussion by joan0909** [sherlock-project/sherlock#971] [Q&A] [answered]
Why instagram is on the list of removed_sites?
Is there any way to add it?

**Comment by iamnotyourenemy** [comment]
Maybe you have to try adding it manually.

**Reply by FrancescaHerbert32** [reply]
how we can add this manually because I [share](https://playapkpro.com/gbwhatsapp-apk-download/) this process on my WhatsApp group

**Comment by ibrahim-gg** [comment]
Cfb

في الأحد، ٢١ فبراير، ٢٠٢١ ١٠:١٤ ص، كتب iamnotyourenemy <
notifications@github.com>:

**Comment by SethFalco** [comment, marked as answer]
Instagram was brought back on 25 Oct 2021 in https://github.com/sherlock-project/sherlock/commit/47c5ccd93e94f10aabe01e6c904a10af0657951f!
